`/components/*` pages are broken

# Bug report

When navigating to a component page, like https://ui.supabase.com/components/icon, there is an error and the page is not loaded

## Describe the bug

The following errors were found on the console : 

```
framework.6fff953eb0f638171baa.js:1 Error: Minified React error supabase/ui#130; visit https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at $i (framework.6fff953eb0f638171baa.js:1:119995)
    at framework.6fff953eb0f638171baa.js:1:68185
    at Io (framework.6fff953eb0f638171baa.js:1:79830)
    at Qu (framework.6fff953eb0f638171baa.js:1:126020)
    at Pi (framework.6fff953eb0f638171baa.js:1:111371)
    at xi (framework.6fff953eb0f638171baa.js:1:111299)
    at _i (framework.6fff953eb0f638171baa.js:1:111162)
    at vi (framework.6fff953eb0f638171baa.js:1:108128)
    at framework.6fff953eb0f638171baa.js:1:57507
    at t.unstable_runWithPriority (framework.6fff953eb0f638171baa.js:1:3875)
```

```
Error: Expected `string` for value, got `undefined`
    at n (%5Bslug%5D-fd042bad38272f37fcbf.js:1:10080)
    at Object.s [as highlight] (%5Bslug%5D-fd042bad38272f37fcbf.js:1:1109431)
    at D.astGenerator (%5Bslug%5D-fd042bad38272f37fcbf.js:1:1427540)
    at bc (%5Bslug%5D-fd042bad38272f37fcbf.js:1:1427660)
    at ro (framework.6fff953eb0f638171baa.js:1:71739)
    at Qu (framework.6fff953eb0f638171baa.js:1:124197)
    at Pi (framework.6fff953eb0f638171baa.js:1:111371)
    at xi (framework.6fff953eb0f638171baa.js:1:111299)
    at _i (framework.6fff953eb0f638171baa.js:1:111162)
    at vi (framework.6fff953eb0f638171baa.js:1:108128)
```

## To Reproduce

Steps to reproduce the behavior, please provide code snippets or a repository:

1. Go to 'https://ui.supabase.com/components/auth'

## Expected behavior

The page should load and display contentn
 
## Screenshots

<img width="1678" alt="Screenshot 2022-01-21 at 17 07 11" src="https://user-images.githubusercontent.com/72619409/150560281-790d7772-ed94-4381-a793-c94f123f4d91.png">

## System information

- OS: macos
- Browser (if applies) Chrome

## Additional context

--- a/components/ComponentIndex.tsx
+++ b/components/ComponentIndex.tsx
@@ -13,10 +13,12 @@ import ButtonBlockRaw from '!!raw-loader!./../_components/button/ButtonBlock'
 import IconBasic from '../_components/icon/IconBasic'
 import IconSize from '../_components/icon/IconSize'
 import IconStroke from '../_components/icon/IconStroke'
+import { IconMail } from '@supabase/ui' // We need to import the icon here aswell, otherwise it will not work in the components above.
 
 import IconBasicRaw from '!!raw-loader!./../_components/icon/IconBasic'
 import IconSizeRaw from '!!raw-loader!./../_components/icon/IconSize'
 import IconStrokeRaw from '!!raw-loader!./../_components/icon/IconStroke'
+import { IconMail as IconMailRaw } from '@supabase/ui' // We need to import the icon here aswell, otherwise it will not work in the components above.
 
 import AuthBasic from '../_components/auth/AuthBasic'
 import AuthSocialProviders from '../_components/auth/AuthSocialProviders'
@@ -368,10 +370,12 @@ RawComponents['BadgeDot'] = BadgeDotRaw
 Components['IconBasic'] = IconBasic
 Components['IconSize'] = IconSize
 Components['IconStroke'] = IconStroke
+Components['IconMail'] = IconMail
 
 RawComponents['IconBasic'] = IconBasicRaw
 RawComponents['IconSize'] = IconSizeRaw
 RawComponents['IconStroke'] = IconStrokeRaw
+RawComponents['IconMail'] = IconMailRaw
 
 // Card
 

